daemon/src/engine/server/sv_init.cpp

```diff
@@ -447,7 +447,7 @@ clients along with it.
 This is NOT called for map_restart
 ================
 */
-void SV_SpawnServer( const char *server )
+void SV_SpawnServer(const std::string pakname, const std::string server)
```

Comment:
Must be an owned string (at some point, so why not here) as the filesystem is cleared in this function invalidating any prior references